src/main/java/ingest/Application.java

```diff
@@ -72,6 +78,11 @@ public static void main(String[] args) {
 	}
 
 	@Bean
+	public Queue updateJobsQueue() {
+		return new Queue(String.format(JobMessageFactory.KAFKA_TOPIC_TEMPLATE, JobMessageFactory.UPDATE_JOB_TOPIC_NAME, SPACE));
```

Comment:
consider renaming this topic template to not contain the word kafka, since it's not specific to kafka.